[#40435] Update SAML documentation

adam-op opened the pull request:
[#40435] Update SAML documentation

https://community.openproject.org/work_packages/40435

Commit:
Update README.md

oliverguenther commented on docs/system-admin-guide/authentication/saml/README.md:
```diff
@@ -132,6 +134,9 @@ Setting.plugin_openproject_auth_saml = Hash(Setting.plugin_openproject_auth_saml
       "name" => "saml",
       "display_name" => "My SSO",
       "assertion_consumer_service_url" => "https:/<YOUR OPENPROJECT HOSTNAME>/auth/saml/callback"
```
Quick reminder: You wanted to add the missing slash here

oliverguenther commented on docs/system-admin-guide/authentication/saml/README.md:
```diff
@@ -61,11 +61,13 @@ saml:
-  # IF your SSL certificate on your SSO is not trusted on this machine, you need to add it here
+  # IF your SSL certificate on your SSO is not trusted on this machine, you need to add it here in ONE line
+  ### one liner to generate certificate in ONE line
+  ### awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' <yourcert.pem>
```
:bow:

adam-op replied:
What does this emoji actually stand for? it seems to have no information to me... at least i cannot combine to anything else than a person with some green shirt

oliverguenther commented on docs/system-admin-guide/authentication/saml/README.md:
```diff
@@ -171,6 +178,24 @@ You may provide attribute names or namespace URIs as follows: `email: ['http://s
+      "email" => "email",
+      "login" => "username",
+      "first_name" => "firstname",
+      "last_name" => "lastname"
```
Even if it works, I'd add two attributes in an array here to show it is possible:

For example:
```
"login" => ['username', 'samAccountName', 'uid'],
```

adam-op replied:
HI @oliverguenther 
i added your suggested changes, please verify them
Thank you

oliverguenther reviewed (changes requested):
Some minor remarks, mostly looking good thanks :+1:

Commit:
minor changes

1) one / was missng
2) added example for array atrributes for the database configuration
